rotkehlchen/db/filtering.py

```diff
@@ -209,6 +209,17 @@ def prepare(self) -> tuple[list[str], list[Any]]:
         return ['tx_hash=?'], [self.tx_hash]
 
 
+@dataclass(init=True, repr=True, eq=True, order=False, unsafe_hash=False, frozen=False)
+class DBOptimismTransactionHashFilter(DBFilter):
```

Comment:
Should I edit the base chain/TransactionHash filter classes for these? I get an ambiguous column name error for the `WHERE (tx_hash = ?)  and (chain_id = ?)` part of the query if I don't edit the filters to specify whether it's the EVM or OP tx_hash/chain_id. 

    statement = 'SELECT DISTINCT EVM.tx_hash, 10, EVM.timestamp, EVM.block_number, EVM.from_address, 
    EVM.to_address, EVM.value, EVM.ga...T JOIN optimism_transactions AS OP ON EVM.tx_hash=OP.tx_hash WHERE 
    (tx_hash=?) AND (chain_id=?) ORDER BY timestamp ASC'
    bindings = ([b'\x06=E\x91\x0f)\xe0\x95JR\xae\xe3\x9f\xeb\xba\x9b\xe7\x84\xd4\x9a\xf7X\x8aY\r\xc2\xfd}\x15kFe', 
    10],)
    
        def execute(self, statement: str, *bindings: Sequence) -> 'DBCursor':
            if __debug__:
                logger.trace(f'EXECUTE {statement}')
            try:
    >           self._cursor.execute(statement, *bindings)
    E           pysqlcipher3.dbapi2.OperationalError: ambiguous column name: tx_hash

I can edit the base filters to always specify `evm_transaction.tx_hash = ?` and the same for chain_id and it'll work without creating the new Optimism___Filter classes, but I don't know if that's acceptable or if it will cause problems for any other chains (although it shouldn't since OP is currently the only chain that deviates from the base EVM query).

Reply:
yes to fix the ambiquity you should just specify the table